catalyst/dl/runner/supervised.py

```diff
@@ -162,6 +164,17 @@ def train(
         if model is not None:
             self.model = model
 
+        if resume is not None:
+            callbacks = callbacks or OrderedDict()
+            checkpoint_callback_flag = all([
```

Comment:
are you sure about `all` instead of `any`?

Reply:
you are right